examples/connectivity/plot_adhd_covariance.py

```diff
@@ -5,10 +5,9 @@
 This example shows how to extract signals from regions defined by an atlas,
 and to estimate a covariance matrix based on these signals.
 """
- 
+n_subjects = 10  # Number of subjects to consider for group-sparse covariance
```

Comment:
I think having `n_subjects=[something]` is useful for having a demo of the parameter.
On my machine (mac), I also have  `n_jobs` as a parameter, and set it to one.  Parallel was crashing Python for me.

I think having `n_jobs=1`, with a comment about tweaking it (including limitations), could be instructive to users.

Or, could remove `n_jobs` and have an example that focuses on the parallel (and perhaps caching) aspects of the package, as kind of a mini-tutorial.